x-pack/test/alerting_api_integration/spaces_only/tests/actions/builtin_action_types/webhook.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { URL, format as formatUrl } from 'url';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+import {
+  getExternalServiceSimulatorPath,
+  ExternalServiceSimulator,
+} from '../../../../common/fixtures/plugins/actions';
+
+// eslint-disable-next-line import/no-default-export
+export default function webhookTest({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const esArchiver = getService('esArchiver');
+  const kibanaServer = getService('kibanaServer');
+
+  async function createWebhookAction(
+    urlWithCreds: string,
+    config: Record<string, string | Record<string, string>> = {}
+  ): Promise<string> {
+    const url = formatUrl(new URL(urlWithCreds), { auth: false });
+    const composedConfig = {
+      headers: {
+        'Content-Type': 'text/plain',
+      },
+      ...config,
+      url,
+    };
+
+    const { body: createdAction } = await supertest
+      .post('/api/action')
+      .set('kbn-xsrf', 'test')
+      .send({
+        name: 'A generic Webhook action',
+        actionTypeId: '.webhook',
+        secrets: {},
+        config: composedConfig,
+      })
+      .expect(200);
+
+    return createdAction.id;
+  }
+
+  describe('webhook action', () => {
+    let webhookSimulatorURL: string = '<could not determine kibana url>';
+
+    // need to wait for kibanaServer to settle ...
+    before(() => {
+      webhookSimulatorURL = kibanaServer.resolveUrl(
+        getExternalServiceSimulatorPath(ExternalServiceSimulator.WEBHOOK)
+      );
+    });
+
+    after(() => esArchiver.unload('empty_kibana'));
+
+    it('webhook can be executed without username and password', async () => {
```

Comment:
This test is in the `spaces_only`  functional test config because it does not use security,  so the webhook simulator can be used without auth.